Makefile

```
@@ -10,8 +10,8 @@ endif
 default: test
 	./$<
 
-test: CFLAGS+=$(TEST_CFLAGS)
-test: test.o encode.o decode.o deps/ok/ok.o
+test: CFLAGS+=$(TEST_CFLAGS) 
```

```suggestion
test: CFLAGS+=$(TEST_CFLAGS)
```

Remove the trailing space?